test/ImportCommand.js

```diff
@@ -69,6 +75,28 @@ describe("ImportCommand", () => {
       }));
     });
 
+    it("works with --max-buffer", (done) => {
```

Comment:
How about a few simple tests for `get .execOpts` in [`test/Command.js`](https://github.com/lerna/lerna/blob/master/test/Command.js#L81) instead? The `.toposort` tests are a good model, and they would be more contextually relevant in that file.

Basically, adding this test to `ImportCommand` is implying that _only_ `ImportCommand` respects it, which is clearly not the case.